test/tests/controls/time_periods/materials/material.i

```diff
@@ -0,0 +1,59 @@
+[Mesh/gen]
+  type = GeneratedMeshGenerator
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[AuxVariables/aux]
+  family = MONOMIAL
+[]
+
+[AuxKernels/mat]
+  type = MaterialRealAux
+  variable = aux
+  property = prop
+[]
+
+[Problem]
+  solve = false
+  material_dependency_check = false
+[]
+
+[Materials]
+  [prop0]
+    type = GenericConstantMaterial
+    prop_names = "prop"
+    prop_values = "1"
+  []
+  [prop1]
+    type = GenericConstantMaterial
+    prop_names = "prop"
+    prop_values = "2"
+    enable = false
```

Comment:
is this the same as setting active/inactive under [Materials]?

Reply:
No, inactive/active happens at parse time. Objects that are inactive because of those are as if they were not in the input file at all.